stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.8.0
+version: 1.8.1
```

Comment:
I'm ok with this, but it is a breaking change. There's a pr in the works for 2.0.0 that will be a major breaking change. Maybe we should apply this there?

Reply:
Do you suggest to merge those changes in with the pr so it can be part of the 2.0.0 release?